index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Survey Form</title>
+	<link rel="stylesheet" href="styles.css">
+</head>
+<body>
+	<center><section class="sec">
+		<div>
+			<h1 id="title">freeCodeCamp Survey Form</h1>
+			<p id="description">Thank you for taking the time to help us improve the platform</p>
+		</div>
+		<div id="d2">
+			<form id="survey-form">
+				<label id="name-label">Name</label><br>
+				<input type="text" name="" id="name" placeholder="Enter your name"><br>
+				<label id="email-label">Email</label><br>
+				<input type="email" name="" id="email" placeholder="Enter your email" required><br>
+				<label id="number-label">Age(Optional)</label><br>
+				<input type="number" name="" id="number" placeholder="Age" required><br>
+				<label id="">Which option best describes your current role?</label><br>
+				<select id="dropdown" placeholder="Select current role">
+					<option>Select current role</option>
+					<option>Student</option>
+					<option>Full time job</option>
+					<option>Full time learner</option>
+					<option>Prefer not to say</option>
+					<option>Other</option>
+				</select><br>
+				<label id="">Would you recommend freeCodeCamp to a friend?</label><br>
+				<input type="radio" name="recommendation" id="number">Definitely<br>
+				<input type="radio" name="recommendation" value="Maybe" id="number">Maybe<br>
+				<input type="radio" name="recommendation" value="Not sure" id="number">Not sure<br>
+				<label id="">What is your favorite feature of freeCodeCamp?</label><br>
+				<select id="dropdown2" placeholder="Select current role">
+					<option>Select an option</option>
+					<option>Challenges</option>
+					<option>Projects</option>
+					<option>Community</option>
+					<option>Open source</option>
+				</select><br>
+				<label id="">What would you like to see improved? (Check all that apply)</label><br>
+				<input type="Checkbox" name="">Front-end Projects<br>
+				<input type="Checkbox" name="">Back-end Projects<br>
+				<input type="Checkbox" name="">Data Visualization<br>
+				<input type="Checkbox" name="">Challenges<br>
+				<input type="Checkbox" name="">Open Source Community<br>
+				<input type="Checkbox" name="">Gitter help rooms<br>
+				<input type="Checkbox" name="">Videos<br>
+				<input type="Checkbox" name="">City Meetups<br>
+				<input type="Checkbox" name="">Wiki<br>
+				<input type="Checkbox" name="">Forum<br>
+				<input type="Checkbox" name="">Additional Courses<br>
+				<label id="">Any comments or suggestions?</label><br>
+				<textarea id="" placeholder="Enter your comment here...">
+					
+				</textarea><br>
+				<button type="submit" name="" id="submit">Submit</button>
+			</form>
+		</div>
```

Comment:
Please Check for your indentation. You code is not clear and this is making your project look messy.

Reply:
- [ ] Good comment. It's clearly talking about the project indentation. It sounds polite & offers a reason for the change request 🥳